frontend/components/site/siteBuilds.jsx

```diff
@@ -206,29 +229,64 @@ function SiteBuilds() {
                   <th scope="col">Actions</th>
+                  { process.env.FEATURE_BUILD_TASKS === 'active' && (
+                    <th className="usa-sr-only" scope="col">Scans</th>
+                  )}
                 </tr>
               </thead>
               <tbody>
                 {builds.data.map((build) => {
-                  const { message, icon } = buildStateData(build);
+                  const { message, done, icon } = buildStateData(build);
 
                   return (
-                    <tr key={build.id}>
+                    <tr key={build.id} className={hasBuildTasks(build) ? 'build-has-scans' : ''}>
                       <th scope="row" data-title="Build">
                         <div className="build-info">
-                          <div className="build-info-icon">
-                            { icon && React.createElement(icon) }
+                          <div className="build-info-prefix">
+                            #
+                            { build.id }
                           </div>
                           <div className="build-info-details">
-                            <h3 className="build-info-status">{ message }</h3>
+                            <h3 className="build-info-status">
+                              { icon && (
+                                <span className="build-info-inline-icon">
+                                  { React.createElement(icon) }
+                                </span>
+                              )}
+                              { message }
+                            </h3>
+                            {build.startedAt && (
+                              <p>
+                                Started
+                                {' '}
+                                <span title={dateAndTimeSimple(build.startedAt)}>
+                                  { timeFrom(build.startedAt) }
+                                </span>
+                              </p>
+                            )}
                             <p>
-                              Build
-                              <b>
-                                #
-                                { build.id }
-                              </b>
+                              { (done && !!build.startedAt) && (
+                                <>
+                                  {message}
+                                  {' '}
+                                  after
+                                  {' '}
+                                  { duration(build.startedAt, build.completedAt) }
+                                </>
+                              )}
+                              {(!done) && (
+                                <>
+                                  {message}
+                                  {' '}
+                                  for
+                                  {' '}
+                                  { duration(build.createdAt, build.completedAt) }
```

Comment:
I'm confused as to why this works (completedAt when not done?) but it does...

Reply:
the second param is not required; if not provided, it uses now.